defaults/main.yml

```diff
@@ -60,6 +60,8 @@ reverse_proxy_image_version: 2.2
 session_management_image_version: latest
 cloudxr_management_image_version: latest
 customization_image_version: latest
+# whether customization image version is newer than 1.2.0
+customization_image_version_gt_1_2_0: "{{ customization_image_version.startswith('latest') or customization_image_version is version('1.2.0', '>') }}"
```

Comment:
portal1 still uses the `v6.3.0` role so we probably don't need this check.
The current customization image version (1.2.0) could also just be pushed again with the new name

Reply:
makes sense. I'm gonna just push it again manually.

Reply:
done.